gasaerexch/plot_gasaerexch_timestep_update.py

```diff
@@ -23,7 +23,36 @@ def __init__(self):
                                                   data.output.qso4_gas_rel, \
                                                   data.output.qsoa_gas_rel
         self.pmid, self.temp, self.dtsubstep = data.input.pmid, data.input.temp,  data.input.dtsubstep
-        
+
+        dtrange = set()
+        for time in data.input.dtsubstep:
+            if time not in dtrange:
```

Comment:
As above (and with the other sets), you don't have to perform this check.

Reply:
Is this line 29 still here?

Reply:
removed now.